invenio_app_ils/records/api.py

```diff
@@ -61,8 +62,35 @@ def create(cls, data, id_=None, **kwargs):
                 document_pid=data[cls.pid_field],
             )
         }
+        if "keywords" not in data:
+            data["keywords"] = []
         return super(Document, cls).create(data, id_=id_, **kwargs)
 
+    def has_keyword(self, keyword):
```

Comment:
In which cases we need this? I mean having keywords expanding inside this array only on search is not enough?

Reply:
I'm not sure if `has_keyword` will ever be useful outside of the class. Since we have `add` and `remove` keyword it made sense to also be able to check if it already includes a keyword.

Reply:
Ok then!